drivers/metadata/postgres/metadata.go

```diff
@@ -34,7 +34,44 @@ func NewReader() func(drivers.DB, ...metadata.ReaderOption) metadata.Reader {
 			}),
 			infos.WithSystemSchemas([]string{"pg_catalog", "pg_toast", "information_schema"}),
 			infos.WithCurrentSchema("CURRENT_SCHEMA"),
-		)
+			infos.WithDataTypeFormatter(func(col metadata.Column) string {
```

Comment:
Can you define this as a standalone function and reference it here? It's a bit big for a lambda.